Detectors/Base/include/DetectorsBase/CTFCoderBase.h

```diff
@@ -129,8 +129,10 @@ class CTFCoderBase
   long getIRFrameSelShift() const { return mIRFrameSelShift; }
```

Comment:
You should not modify CTFCoderBase, which serves all detectors, for adding extra option used by CTP only. This should go to the CTP CTFCoder

Reply:
But there is already CTP stuff related to mBCShift.
I 'd like to introduce other global shift for inputs while mBCShits is used for class mask as until now.
So I should create  in CTFCoder.h method
  bool finaliseCCDB(o2::framework::ConcreteDataMatcher& matcher, void* obj);
and add my stuff there ?

Reply:
`mBCShift` corresponds to `TriggerOffsetsParam::customOffset...` which has an entry per detector. On top of that you want to add an extra shift used by the CTP only. So, yes, you should add an extra data member and getter/setter specific for the CTP CTFCoder.h.

Reply:
done